stubs/docker/docker/models/containers.pyi

```diff
@@ -113,9 +116,92 @@ class ContainerCollection(Collection[Container]):
+        device_read_iops: Incomplete | None = None,
+        device_write_bps: Incomplete | None = None,
+        device_write_iops: Incomplete | None = None,
+        devices: list[str] | None = None,
+        device_requests: list[DeviceRequest] | None = None,
+        dns: list[Incomplete] | None = None,
+        dns_opt: list[Incomplete] | None = None,
+        dns_search: list[Incomplete] | None = None,
+        domainname: str | list[Incomplete] | None = None,
+        entrypoint: str | list[str] | None = None,
+        environment: dict[str, str] | list[str] | None = None,
+        extra_hosts: dict[str, str] | None = None,
+        group_add: list[str | int] | None = None,
+        healthcheck: dict[Incomplete, Incomplete] | None = None,
+        hostname: str | None = None,
+        init: bool | None = None,
+        init_path: str | None = None,
+        ipc_mode: str | None = None,
+        isolation: str | None = None,
+        kernel_memory: str | int | None = None,
+        labels: dict[str, str] | list[str] | None = None,
+        links: dict[str, str | None] | None = None,
+        log_config: LogConfig | None = None,
+        lxc_conf: dict[Incomplete, Incomplete] | None = None,
+        mac_address: str | None = None,
+        mem_limit: str | int | None = None,
+        mem_reservation: str | int | None = None,
+        mem_swappiness: int | None = None,
+        memswap_limit: str | int | None = None,
+        mounts: list[Mount] | None = None,
+        name: str,
+        nano_cpus: int | None = None,
+        network: str | None = None,
+        network_disabled: bool = False,
+        network_mode: str | None = None,
+        networking_config: dict[str, EndpointConfig] | None = None,
+        oom_kill_disable: bool = False,
+        oom_score_adj: int | None = None,
+        pid_mode: str | None = None,
+        pids_limit: int | None = None,
+        platform: str | None = None,
+        ports: dict[str, int | None] | None = None,
+        privileged: bool = False,
+        publish_all_ports: bool = False,
+        read_only: bool | None = None,
+        restart_policy: dict[Incomplete, Incomplete] | None = None,
+        runtime: str | None = None,
+        security_opt: list[str] | None = None,
+        shm_size: str | int | None = None,
+        stdin_open: bool = False,
+        stop_signal: str | None = None,
+        storage_opt: dict[Incomplete, Incomplete] | None = None,
+        stream: bool = False,
+        sysctls: dict[Incomplete, Incomplete] | None = None,
+        tmpfs: dict[str, str] | None = None,
+        tty: bool = False,
+        ulimits: list[Ulimit] | None = None,
+        use_config_proxy: bool | None = None,
+        user: str | int | None = None,
+        userns_mode: str | None = None,
+        uts_mode: str | None = None,
+        version: str,
+        volume_driver: str | None = None,
+        volumes: dict[str, str] | list[str] | None = None,
```

Comment:
I think this annotation should be `dict[str, dict[str, str]]`.

According to the documentation

```
volumes (dict or list) –

A dictionary to configure volumes mounted inside the container. The key is either the host path or a volume name, and the value is a dictionary with the keys:

   - bind The path to mount the volume inside the container
   - mode Either rw to mount the volume read/write, or ro to mount it read-only.

For example:

{'/home/user1/': {'bind': '/mnt/vol2', 'mode': 'rw'},
 '/var/www': {'bind': '/mnt/vol1', 'mode': 'ro'}}
```